dspace-server-webapp/src/test/java/org/dspace/app/rest/RequestItemRepositoryIT.java

```diff
@@ -533,29 +533,6 @@ public void testPutBadRequest()
 
         ObjectWriter mapperWriter = new ObjectMapper().writer();
 
-        // Unauthenticated user
```

Comment:
@mwoodiupui : Rather than just removing all the unauthenticated user tests, I'd recommend we add a single unauthenticated user test which *proves* that only the token is required.  

So, I'd expect that we'd change one of these to prove that a successful response is returned.  That way we have a test to prove that our `permitAll()` is working properly...and that test would fail if someone ever tries to change that `permitAll()` to something else.

Reply:
Good idea.  I think this is done.